app/api/services/abr.py

```diff
@@ -29,7 +29,7 @@ def call_abr_api(self, url):
         try:
             response = requests.get(url)
             if response.ok:
-                error = self.get_abr_exception(response.content)
+                error = self.get_abr_exception(response.content.decode())
```

Comment:
@tobiasbrown not sure if decode() requires here

Reply:
This is needed!